.github/workflows/build-pgaf-community-nightlies.yml

```diff
@@ -0,0 +1,79 @@
+name: Build and publish pgaf nightly packages
```

Comment:
very minor, to distinguish from enterprise one:

```suggestion
name: Build and publish pgaf community nightly packages
```

Reply:
Fixed